src/main/kotlin/org/kamiblue/client/gui/rgui/component/StringButton.kt

```diff
@@ -1,12 +1,16 @@
 package org.kamiblue.client.gui.rgui.component
 
+import org.kamiblue.client.module.modules.client.ClickGUI
 import org.kamiblue.client.setting.settings.impl.primitive.StringSetting
 import org.kamiblue.client.util.math.Vec2f
 import org.lwjgl.input.Keyboard
 import kotlin.math.max
 
 class StringButton(val setting: StringSetting) : BooleanSlider(setting.name, 1.0, setting.description, setting.visibility) {
 
+    override val isBold
+        get() = setting.isModified and ClickGUI.showModifiedInBold
```

Comment:
```suggestion
    override val isBold
        get() = setting.isModified && ClickGUI.showModifiedInBold
```

Reply:
I will do this, however is there any benefit to this or is this just stylistic?

Reply:
It's faster on performance

Reply:
Thanks

Reply:
> It's faster on performance, and in some cases `and` can have different results than `&&` if I'm reading the docs correctly 

It is not faster. It is just some style stuff and operator precedence stuff